### ISSUE TITLE
I want to ask you the something about  the data

### ISSUE THREAD

**Issue by tlament** [ColaLL/IONE#2]
I view the code and want to konw where is the anchor nodes, 0-1609 is the anchor nodes?

**Comment by ColaLL** [comment]
yes (0-1608), pls note that the nodes' numbers which bigger than 1608 are not anchor pairs even they have the same numbers.

**Comment by ColaLL** [comment]
> I view the code and want to konw where is the anchor nodes, 0-1609 is the anchor nodes?

yes (0-1608), pls note that the nodes' numbers which bigger than 1608 are not anchor pairs even they have the same numbers.

**Comment by tlament** [comment]
I get it,Thank you very much.

**Comment by githubzcl** [comment]
I run IONE.java by using my dataset,but I don't find embeddings file in the folder "embeddings".How to get them？
